### ISSUE TITLE
kaiju

### ISSUE THREAD

**Issue by yolanda-huang** [ubccr/software-layer#183]
https://github.com/bioinformatics-centre/kaiju

**Comment by tonykew** [comment]
Test build of Kaiju version 1.7.3 complete - more info to follow

**Comment by tonykew** [comment]
Kaiju version 1.7.3 has been published to the ccrsoft/2023.01 software release


```
vortex1$ module spider kaiju
----------------------------------------------------------------------------
  kaiju: kaiju/1.7.3-Python-3.9.6
----------------------------------------------------------------------------
    Description:
      Kaiju is a program for sensitive taxonomic classification of
      high-throughput sequencing reads from metagenomic whole genome
      sequencing experiments

    Properties:
      Bioinformatic libraries/apps

    You will need to load all module(s) on any one of the lines below before the "kaiju/1.7.3-Python-3.9.6" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
vortex1$ module load gcc/11.2.0 openmpi/4.1.1 kaiju/1.7.3-Python-3.9.6
vortex1$ 
vortex1$ ls ${EBROOTKAIJU}/bin
kaiju                kaiju-convertNR                kaiju-mkbwt
kaiju2krona          kaiju-excluded-accessions.txt  kaiju-mkfmi
kaiju2table          kaiju-gbk2faa.pl               kaijup
kaiju-addTaxonNames  kaiju-makedb                   kaiju-taxonlistEuk.tsv
kaiju-convertMAR.py  kaiju-mergeOutputs             kaijux
vortex1$ 
vortex1$ kaiju -h
Kaiju 1.7.3
Copyright 2015-2020 Peter Menzel, Anders Krogh
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

Usage:
   kaiju -t nodes.dmp -f kaiju_db.fmi -i reads.fastq [-j reads2.fastq]
[...]
vortex1$ 

```
